File: app/src/main/java/com/example/msdc/api/PersonDetails.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class PersonDetails {
    @SerializedName("id")
    int id;

    @SerializedName("name")
    String name;

    @SerializedName("biography")
    String biography;

    @SerializedName("popularity")
    int popularity;

    @SerializedName("place_of_birth")
    String placeOfBirth;

    @SerializedName("profile_path")
    String profilePath;

    @SerializedName("adult")
    boolean adult;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getBiography() {
        return biography;
    }

    public int getPopularity() {
        return popularity;
    }

    public String getPlaceOfBirth() {
        return placeOfBirth;
    }

    public String getProfilePath() {
        return profilePath;
    }

    public boolean isAdult() {
        return adult;
    }
}

File: app/src/main/java/com/example/msdc/api/PersonDetails.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class PersonDetails (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("name") val name: String? = null,
    @field:SerializedName("biography") val biography: String? = null,
    @field:SerializedName("popularity") val popularity: Int? = 0,
    @field:SerializedName("place_of_birth") val placeOfBirth: String? = null,
    @field:SerializedName("profile_path") val profilePath: String? = null,
    @field:SerializedName("adult") val isAdult: Boolean? = false
) : Parcelable